### ISSUE TITLE
Possibility to disable and hide (!) all Thumbnails settings in TopMenu

### ISSUE THREAD

**Issue by cmahnke** [ProjectMirador/mirador#3312]
Currently Thumbnails can be deactivated, see [here](https://github.com/ProjectMirador/mirador/blob/d61b13a505e351f3f19cb2e68aba54fbbe3e02cf/src/config/settings.js#L299), but the user can reenable them, which is only desirable im Mirador is able to generate them on it's own. ;)

But in case they aren't defined in the manifest you'll get just some broken icons for now.

Setting [allowTopMenuButton](https://github.com/ProjectMirador/mirador/blob/d61b13a505e351f3f19cb2e68aba54fbbe3e02cf/src/config/settings.js#L259) to false doesn't really help either, since this would also disable desired functionality, like view switching.

Here is a screenshot to illustrate the desired setting, the last row should be hidden on demand.

![Bildschirmfoto 2020-10-28 um 21 11 27](https://user-images.githubusercontent.com/194820/97492042-4e3cdb80-1963-11eb-94c1-795a91b5722f.png)

**Comment by cmahnke** [comment]
I tried to get rid of my requirement by providing thumbnails, but it's not as easy as the IIIF specification implies, at least not with current Mirador 3. To give you some context, I'm only using static images, no image or tile server at all. But that fact isn't obvious from my manifest.

Anyways, even if I try to provide a static thumbnail as described in the [example of the specification](https://iiif.io/api/presentation/2.1/#canvas) by providing a `thumbnail'`entry, I can see that Mirador constructs requests using the service definition of the image and ignoring the `thumbnail` entry. 

Is this behaviour expected or even intended?

**Comment by mejackreed** [comment]
Could you please provide a sample manifest so we can take a look? Thanks!

**Comment by cmahnke** [comment]
If you don't expect the URLs to be resolvable it's not a problem. This example should be enought to witness the behavior in a browser, note, that the constructed URLs depend on the API level:
https://gist.githubusercontent.com/cmahnke/da432a6fcdf6675231aa4ba087595d65/raw/d5ebda6f5beb829ebc17f0d2e43ed2a7fdd9e3e9/index.json

**Comment by mejackreed** [comment]
Great thank you. Could you also provide one of the canvas image resources info.json responses?

**Comment by mejackreed** [comment]
In looking at this manifest it seems that the image services indicate support level1 compliance. https://iiif.io/api/image/2.1/compliance/#region

I think this is what is causing Mirador some difficulties. Because you are generating static tiles, is it possible that there is not support for level1? What happens if you change that to level0?

**Comment by cmahnke** [comment]
Yes, but these won't resolve to anything as well (yet). I'm using [Libvips](https://github.com/libvips/libvips) to create those.
See https://gist.githubusercontent.com/cmahnke/da432a6fcdf6675231aa4ba087595d65/raw/39d7ec4b0ebeafaa7b291e174b19210c3ecbc84f/info.json

**Comment by mejackreed** [comment]
Thanks.. so it seems like the manifest says this is level1, though it is only a level0? Can you modify the manifests to be level0?

**Comment by cmahnke** [comment]
Yes, as I already wrote, I already did that and it's of no use. Actually the constructed requests just change, not for the better. It tries two URLs per image (from the example):
* `http://localhost:1313/post/1x1-des-tapezierens/page001/`
* `http://localhost:1313/post/1x1-des-tapezierens/page001`

The difference is just the slash at the end, it's just the `@id` of the image.

See: 
https://gist.githubusercontent.com/cmahnke/da432a6fcdf6675231aa4ba087595d65/raw/d197f805006222d2bc00887287acffd6d9895c44/index-2.json

But anyways, shouldn't Mirador just request the `@id` in the `thumbnail` section, regardless of the API level?
If it's not the default, can I force it somehow?

**Comment by cmahnke** [comment]
I've tracked it down and created a separate issue: #3330

**Comment by mejackreed** [comment]
Thanks @cmahnke we will take a look 👍
